src/SvelteTable.svelte

```diff
@@ -45,17 +58,21 @@
     }
   };
 
-  $: sortedRows = rows.sort((a, b) => {
-    if (typeof a[sortBy] === 'string' || typeof b[sortBy] === 'string')
-      return sortStrings(a[sortBy], b[sortBy]);
-    if (a[sortBy] > b[sortBy]) {
-      return sortOrder;
-    } else if (a[sortBy] < b[sortBy]) {
-      return -sortOrder;
-    }
+  const sortRows = (rows, sortOrder, from, to) => {
+    const rowsSorted = rows.sort((a, b) => {
```

Comment:
I think sortedRows reads better here

Reply:
Also .sort doesn't return a sorted copy of an array but a reference to the object itself. So sortedRows equals rows